rune/libenclave/internal/runtime/pal/skeleton/sgxsign.c

```diff
@@ -152,7 +152,7 @@ static bool mrenclave_ecreate(EVP_MD_CTX *ctx, uint64_t blob_size)
 
 	memset(&mrecreate, 0, sizeof(mrecreate));
 	mrecreate.tag = MRECREATE;
-	mrecreate.ssaframesize = 1;
+	mrecreate.ssaframesize = sgx_calc_ssaframesize(miscselect, xfrm);
```

Comment:
If ssa frame is larger than one page, the ssa frame space should be enlarged as well. Refer to:
```
        .section ".data", "aw"

encl_ssa:
        .space 4096
```